src/mds/CInode.h

```diff
@@ -601,7 +601,11 @@ class CInode : public MDSCacheObject, public InodeStoreBase, public Counter<CIno
 
   inodeno_t ino() const { return inode.ino; }
   vinodeno_t vino() const { return vinodeno_t(inode.ino, last); }
+  #ifndef _WIN32
   int d_type() const { return IFTODT(inode.mode); }
+  #else
+  int d_type() const { return -ENOTSUP; }
+  #endif
```

Comment:
NAK on this. I suggest adding a define for IFTODT if it does not exist.

Reply:
Indeed, adding IFTODT is probably better than stubbing it, even though it's not used on Windows at the moment.

My main concern was that the Windows stat function doesn't properly set the file mode for some file types (e.g. symlinks).